test/service/test_request_validation.py

```diff
@@ -352,3 +330,14 @@ def test_order(self):
                     self.assertBadRequest(url, arg)
                 else:
                     self.assertResponseStatus(url, 200)
+
+    def test_version_param(self):
+        url = self.base_url.set(path='repository/files/74897eb7-0701-4e4f-9e6b-8b9521b2816b',
+                                query_params={'version': 'foo'})
+        self.assertBadRequest(url, 'Invalid value for `version`')
+
+        with mock.patch('azul.service.repository_controller.RepositoryController.download_file',
+                        new=lambda *args, **kwargs: {'Status': 302, 'Location': 'foo'}):
+            url = self.base_url.set(path='fetch/repository/files/74897eb7-0701-4e4f-9e6b-8b9521b2816b',
+                                    query_params={'version': '2018-11-02T11:33:44.450442Z'})
+            self.assertResponseStatus(url, 200)
```

Comment:
```suggestion
            self.assertResponseStatus(url, 404)
```

proves that version passed validation